server/src/main/resources/schema/wildfly-config_3_0.xsd

```diff
@@ -2127,6 +2127,15 @@
                 </xs:choice>
                 <xs:attribute name="console-enabled" type="xs:boolean" use="optional" default="true"/>
                 <xs:attribute name="http-upgrade-enabled" type="xs:boolean" use="optional" default="false"/>
+                <xs:attribute name="allowed-origins" type="xs:string" use="optional" >
```

Comment:
I would prefer it if one of the list types was used for the attribute type which would make it a space separated list.  Any form of comma separated list requires reading of the documentation to understand how to format the content meaning tools can not do this automatically.

We already have a urlListType and stringListType in the schema that should be suitable for this.